healsparse/healSparseMap.py

```diff
@@ -1257,6 +1260,8 @@ def degrade(self, nside_out, reduction='mean'):
                 raise NotImplementedError('Cannot degrade a wide_mask map with this \
                 reduction operation, try and/or')
             else:
+                if weights is not None:
+                    raise Warning('Ignoring weights')
```

Comment:
This warning should say why we're ignoring the weights (can't use for wide mask).